script.js

```diff
@@ -1,4 +1,88 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+// Player One clicks submit
+// Roll two dices and show value
+// Player One chooses dice order
+// Player Two clicks submit
+// Roll two dices and show value
+// Player Two chooses dice order
+// Evaluate higher combined score and output result. Reset game
+
+var gameMode = "playerOneRoll";
+var dices = [];
+var playerOneScore = "";
+var playerTwoScore = "";
+
+var main = function(input) {
+  if (gameMode == "playerOneRoll") {
+    var rollPlayerOne = rollPlayer();
+    gameMode = "playerOneChooses";
+    return `Player One has rolled: <br> 1st Dice: ${rollPlayerOne[0]} <br> 2nd Dice: ${rollPlayerOne[1]}`;
+  }
+  if (gameMode == "playerOneChooses") {
+    if (input != 1 && input != 2) {
+      return `Please enter either 1 or 2 to choose which dice goes first.`
+    } else {
+      gameMode = "playerTwoRoll";
+      playerOneScore = combineScore(input);
+      return `Player One score: ${playerOneScore}`;
+      }
+  }
+  if (gameMode == "playerTwoRoll") {
+    var rollPlayerTwo = rollPlayer();
+    gameMode = "playerTwoChooses";
+    return `Player Two has rolled: <br> 1st Dice: ${rollPlayerTwo[0]} <br> 2nd Dice: ${rollPlayerTwo[1]}`;
+  }
+  if (gameMode == "playerTwoChooses") {
+    if (input != 1 && input != 2) {
+      return `Please enter either 1 or 2 to choose which dice goes first.`
+    } else {
+      gameMode = "compare";
+      playerTwoScore = combineScore(input);
+      return `Player Two score: ${playerTwoScore}`;
+      }
+  }
+  if (gameMode == "compare") {
+    return compareScore()
+  }
+}
+
+//Helper function to get a random number from 1-6
```

Comment:
Good job using comments to describe what each function does! Keep up this great habit!